src/NuGet.Services.EndToEnd/SearchResultTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using NuGet.Services.EndToEnd.Support;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace NuGet.Services.EndToEnd
+{
+    public class SearchResultTests : IClassFixture<TrustedHttpsCertificatesFixture>
+    {
+        private readonly ITestOutputHelper _logger;
+        private readonly Clients _clients;
+
+        public SearchResultTests(ITestOutputHelper logger)
+        {
+            _clients = Clients.Initialize();
+            _logger = logger;
+        }
+        
+        [SemVer2Fact]
+        public async Task RegistrationValuesInResultMatchRequestSemVerLevel()
+        {
+            // Arrange
```

Comment:
Technically, we need to push both a semver 1 and semver 2 package here. They should be the same as the ones the other test use.

Imagine running this on a completely new environment.